plugins/funind/g_indfun.mlg

```diff
@@ -159,7 +159,7 @@ GRAMMAR EXTEND Gram
   GLOBAL: function_rec_definition_loc ;
 
   function_rec_definition_loc:
-    [ [ g = Vernac.rec_definition -> { Loc.tag ~loc g } ]]
+    [ [ g = Vernac.fix_definition -> { Loc.tag ~loc g } ]]
```

Comment:
I think you mean `function_rec_definition_loc`?  Yes, that could be changed.  I didn't touch it yet because the documentation doesn't show that nonterminal name--doc_grammar replaces uses of the nonterminal with its definition (aka splicing).  There are 100-200 such nonterminals that doc_grammar splices out to create the documentation version of the grammar.

Reply:
> I think you mean function_rec_definition_loc?

Yes.

Reply:
Done.